subsys/bluetooth/host/hci_core.c

```diff
@@ -1904,6 +1904,16 @@ static void le_conn_update_complete(struct net_buf *buf)
 	LOG_DBG("status 0x%02x %s, handle %u",
 		evt->status, bt_hci_err_to_str(evt->status), handle);
 
+	if (!IN_RANGE(sys_le16_to_cpu(evt->interval), BT_HCI_LE_INTERVAL_MIN, BT_HCI_LE_INTERVAL_MAX)) {
```

Comment:
I don't think we should test these values if `evt->status != 0`.

We could consider moving these checks to after we have assigned them to the conn (since this change won't prevent that anyways), so that we don't need to do the `sys_le16_to_cpu` parts twice as well

Reply:
Done. Thank you.